synapse/appservice/__init__.py

```diff
@@ -32,38 +32,6 @@ class ApplicationServiceState:
     UP = "up"
 
 
-class AppServiceTransaction:
```

Comment:
Moving this makes the diff rather difficult to follow. It looks like the difference is the `__init__` takes an `ephemeral` and pipes it into `push_bulk`?

Reply:
It also takes an `ApplicationService`